packages/astro/src/content/vite-plugin-content-virtual-mod.ts

```diff
@@ -29,34 +33,41 @@ export function astroContentVirtualModPlugin({
 
 	const contentEntryConfigByExt = getContentEntryConfigByExtMap(settings);
 	const contentEntryExts = [...contentEntryConfigByExt.keys()];
+	const dataEntryExts = getDataEntryExts(settings);
 
-	const extGlob = getExtGlob(contentEntryExts);
-	const entryGlob = `${relContentDir}**/*${extGlob}`;
 	const virtualModContents = fsMod
 		.readFileSync(contentPaths.virtualModTemplate, 'utf-8')
+		.replace(
+			'@@COLLECTION_NAME_BY_REFERENCE_KEY@@',
+			new URL('reference-map.json', contentPaths.cacheDir).pathname
+		)
 		.replace('@@CONTENT_DIR@@', relContentDir)
-		.replace('@@ENTRY_GLOB_PATH@@', entryGlob)
-		.replace('@@RENDER_ENTRY_GLOB_PATH@@', entryGlob);
+		.replace('@@CONTENT_ENTRY_GLOB_PATH@@', `${relContentDir}**/*${getExtGlob(contentEntryExts)}`)
+		.replace('@@DATA_ENTRY_GLOB_PATH@@', `${relContentDir}**/*${getExtGlob(dataEntryExts)}`)
```

Comment:
I see that this string is repeated... one more reason to move it inside a constant somewhere! (avoid typos)

Reply:
Per comment above, we can't hoist these to a `consts` file due to Vite's limitations 😕